elasticsearch/prep-install.origin

```diff
@@ -0,0 +1,15 @@
+#reconcile upstream to downstream
+es_source=/usr/share/elasticsearch
+for file in $(ls $es_source)
```

Comment:
I wasn't sure how to proceed about this suggestion so I created a pull request into this pull request https://github.com/jcantrill/origin-aggregated-logging/pull/1